multi-agent/config/app_config.py

```
@@ -1,6 +1,8 @@
 import secrets
+from typing import List
```

Ruff (0.15.20)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)